config/delta-producer/dump-file-publisher/config.json

```diff
@@ -1,66 +1,50 @@
 {
-  "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/deltaDumpFileCreation/administrative-units": {
+  "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/deltaDumpFileCreation/administrative-units" : {
     "dcatDataSetSubject": "http://data.lblod.info/datasets/delta-producer/dumps/AdministrativeUnitsCacheGraphDump",
     "targetGraph": "http://redpencil.data.gift/id/deltas/producer/administrative-units",
     "fileBaseName": "dump-administrative-units",
-    "targetDcatGraph": "http://mu.semte.ch/graphs/public",
-    "targetFilesGraph": "http://mu.semte.ch/graphs/public",
-    "cleanupOldDumps": true
+    "compression": true
```

Comment:
For this file, we should probably keep the previous config ? From what I see here https://github.com/lblod/delta-producer-dump-file-publisher/pull/3 compression is not a thing anymore as we clean up old dumps automatically